site/en/r2/tutorials/load_data/csv.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>Please use 'relu' instead of tf.keras.backend.relu.</p><p><br></p><p>Also you can do `x = tf.keras.Input(shape=(input_dim, ))`.</p><p><br></p><p>&gt; hidden_units: [int] the layer sizes of the DNN (input layer first)</p><p><br></p><p>hidden_units: (int) the layer sizes of the DNN (input layer first)</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>Done.</p><p><br></p><p>But it should be square brackets, I think, as it is a list of ints, not an int.</p>

Reply:
i think this is right currently